content/SCALE/SCALETutorials/Network/Interfaces/_index.md

```diff
@@ -14,105 +15,203 @@ tags:
 
+## Editing an Interface
+To change an existing interface, select the existing interface listed on the **Interfaces** widget and click **Edit** to open the **Edit Interface** screen.
+
+The **Edit Interface** and **Add Interface** settings are identical but the **Type** and **Name** fields are not editable after saving an interface.
 **Name** shows on the **Edit Interface** screen, but you cannot change the name.
-**Type** only shows on the **Add Interface** screen. You cannot edit these settings after saving the interface.
-If you made a mistake with either field and clicked **Save**, the only option to change either setting is to delete the interface and create a new one with the desired name and type.
+**Type** only shows on the **Add Interface** screen.
+If you make a mistake with either and clicked **Save**, the only option to change either setting is to delete the interface and create a new one with the desired name and type.
 
-If you want to change from a DHCP-provided IP address to a static IP, you must also add the new default gateway and DNS nameservers that work with the new IP address.
-See [Setting Up a Static IP]({{< relref "SettingUpStaticIPs.md" >}}) for more information.
+To change from a DHCP-provided IP address to a static IP, you must also add the new default gateway and DNS name servers that work with the new IP address.
 
-After making the changes, click **Save**.
+Click **Save** after making all changes.
 
-Test the change as described in [Testing Network Interface Changes](#testing-network-interface-changes).
+Test the change as described  above in [Testing Network Interface Changes](#testing-network-interface-changes).
 
+## Deleting an Interface
 {{< hint type=warning >}}
-If you delete the primary network interface you can lose your TrueNAS connection and the ability to communicate with the TrueNAS through the web interface!
+Deleting the primary network interface can result in lost access to the TrueNAS UI and losing the connection to TrueNAS!
+
+If saved changes cause lost access to the UI, you might need command line knowledge and/or physical access to the TrueNAS system to fix misconfigured network settings.
 
-If you save the changes you lose access to the UI, you might need command line knowledge or physical access to the TrueNAS system to fix misconfigured network settings.
+Do not delete the primary network interface!
 {{< /hint >}}
 
-## Deleting an Interface
 Click the <i class="material-icons" aria-hidden="true" title="delete">delete</i> icon for the interface.
 A delete interface confirmation dialog opens.
 
-{{< hint type=warning >}}
-Do not delete the primary network interface!
+## Adding Static or Alias IP Addresses
+TrueNAS allows adding static IP addresses to an interface and using them as aliases.
+Static IP addresses set a fixed address for an interface that external devices or websites need to access or remember, such as for VPN access.
```

Comment:
I'm not sure if the next few sentences of discription are relevant to the way TrueNAS uses "alias" in the UI (at least not how it uses it when assigning an alias/static IP for the primary interface). I don't know that we need to be too detailed in explaining here, just that this is the procedure for setting a fixed IP address (like the first 1.5 sentences here) on your interface.

Reply:
the request for this change was to be more clear/detailed in explaining this. I didn't think our existing content was far off the mark, but apparently, there was "confusion" about aliases versus static IP addresses.